lib/Listener/UserCreatedListener.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
```suggestion
<?php

declare(strict_types=1);

```

and copyright header

Reply:
Added copyright